docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -1116,6 +1133,28 @@ public Consumer<Message<String>> consume2() {
 }
 ----
 
+===== Resource Provision
+Servicebus topic, queue and subscription could be auto create by add such properties:
```

Comment:
Let's write it as "Service bus binder supports provisioning of queue, topic and subscription, users could use below properties to enable provisioning."